### PR TITLE
Update uglify-es notice

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,7 +130,11 @@ UglifyJS does not currently support any ES6 syntax, so if you are using Uglify t
 
 To prevent these errors - specify the uglify option, which will enable all plugins and, as a result, fully compile your code to ES5. However, the `useBuiltIns` option will still work as before, and only include the polyfills that your target(s) need.
 
-> NOTE: Uglify has a work-in-progress "Harmony" branch to address the lack of ES6 support, but it is not yet stable.  You can follow its progress in [UglifyJS2 issue #448](https://github.com/mishoo/UglifyJS2/issues/448).  If you require an alternative minifier which _does_ support ES6 syntax, we recommend using [Babili](https://github.com/babel/babili).
+> NOTE: Uglify has a complete and stable alternative (https://npmjs.org/package/uglify-es). However it does not support all ES2015+ syntax. If you require an alternative minifier which _does_ support all babel syntax, we recommend using [Babili](https://github.com/babel/babili).
+
+### `targets.uglifyes`
```
Comment (yavorsky) [inline README.md]: Actually we don't have this option.

What ES2015+ features uglify-es doesn't support ? If something that can be covered by the plugin, then yes, we can somehow store (hardcode) this coverage info and enable just plugins we need. But also we'll have to track this repo and format our data to match features they already finished.

If uglify-es support all of es2015+ things, it won't affect the result at all (disable all plugins).

File: README.md [hunk 2]
```diff
@@ -130,7 +130,11 @@ UglifyJS does not currently support any ES6 syntax, so if you are using Uglify t
 
 To prevent these errors - specify the uglify option, which will enable all plugins and, as a result, fully compile your code to ES5. However, the `useBuiltIns` option will still work as before, and only include the polyfills that your target(s) need.
 
-> NOTE: Uglify has a work-in-progress "Harmony" branch to address the lack of ES6 support, but it is not yet stable.  You can follow its progress in [UglifyJS2 issue #448](https://github.com/mishoo/UglifyJS2/issues/448).  If you require an alternative minifier which _does_ support ES6 syntax, we recommend using [Babili](https://github.com/babel/babili).
+> NOTE: Uglify has a complete and stable alternative (https://npmjs.org/package/uglify-es). However it does not support all ES2015+ syntax. If you require an alternative minifier which _does_ support all babel syntax, we recommend using [Babili](https://github.com/babel/babili).
```
Comment (yavorsky) [inline README.md]: As for me, [uglify-es branch link from github](https://github.com/mishoo/UglifyJS2/tree/harmony) is better than npm package link. We don't need to thorw users around different resources. Also, github README has npm install instructions. It's all we need.

File: README.md (hunk 1 above)
Comment (graingert) [inline README.md]: oh woops I didn't mean to add this in this branch.

Comment (graingert) [inline README.md]: See also: https://github.com/kangax/compat-table/issues/1120 and https://github.com/kangax/compat-table/issues/1120

Comment (yavorsky) [inline README.md]: @graingert yeah, it would be nice!

File: README.md (hunk 2 above)
Comment (existentialism) [inline README.md]: Also, "complete and stable alternative" reads weird when it's followed by "However it does not support all".

Maybe something like:

> Uglify has support for ES2015 syntax via [uglify-es](https://github.com/mishoo/UglifyJS2/tree/harmony). If you are using syntax unsupported by `uglify-es`, we recommend using [Babili](https://github.com/babel/babili).

**Pull request closed without merging**